pandas/tests/test_index.py

```diff
@@ -3324,6 +3332,22 @@ def setUp(self):
     def create_index(self):
         return date_range('20130101', periods=5)
 
+    def test_shift(self):
+        # test shift for datetimeIndex and non datetimeIndex
+        # err8083
+
+        drange = self.create_index()
```

Comment:
this is ONLY true for DatetimeIndex. Need separate expectation for TimedeltaIndex/PeriodIndex (IOW separate methods).